sky/utils/schemas.py

```diff
@@ -58,9 +82,14 @@ def _get_single_resources_schema():
             'use_spot': {
                 'type': 'boolean',
             },
+            # Deprecated: use 'job_recovery' instead. This is for backward
+            # compatibility, and can be removed in 0.7.0.
```

Comment:
```suggestion
            # compatibility, and can be removed in 0.8.0.
```
we decided to deprecate it in 0.8.0 iiuc?